pkg/action.go

```diff
@@ -34,7 +34,7 @@ func checkError(err error) {
 	}
 }
 
-func powerStateChange(clientSet *hiveclient.Clientset, clusterDeployment *hivev1.ClusterDeployment, powerState string) string {
+func powerStatePatch(clientSet *hiveclient.Clientset, clusterDeployment *hivev1.ClusterDeployment, powerState string) string {
```

Comment:
Seems this function is now unused?

Reply:
Yeah, as per the comment below, holding on to it for now.

Reply:
To not affect existing use cases, and meet the original RBAC, I started using Patch for the bulk updates.